benchmarks/abstract_test_type.py

```
@@ -0,0 +1,132 @@
+    those keys and injecting them into the test. For example, if
+    benchmark_config.json contains a line `"spam" : "foobar"` and a subclasses X
+    passes an argument list of ['spam'], then after parsing there will
+    exist a member `X.spam = 'foobar'`.
+    '''
+
+    def __init__(self,
+                 config,
+                 name,
+                 requires_db=False,
+                 accept_header=None,
+                 args=[]):
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)